modules/oxide/src/less/theme/content-ui.less

```
@@ -24,5 +24,6 @@
 @import 'content/table-of-contents/table-of-contents';
 @import 'content/transparent/transparent';
 @import 'content/table/table';
+@import 'content/upploadcare/upploadcare';
```

typo in the folder and filename
```suggestion
@import 'content/uploadcare/uploadcare';
```